source/unit/Fixture.ooc

```diff
@@ -52,7 +52,7 @@ Fixture: abstract class {
 		this add(test)
 	}
 	run: func -> Bool {
-		failures := VectorList<TestFailedException> new()
+		failures := VectorList<TestFailedException> new(32, false)
```

Comment:
`32` just because you really want to set the second argument to `false`, right ?